packages/aws-cdk/lib/api/deployments.ts

```diff
@@ -427,6 +436,7 @@ export class Deployments {
       deployName: options.deployName,
       quiet: options.quiet,
       ci: options.ci,
+      exitOnConfigComplete: options.exitOnConfigComplete,
```

Comment:
Where is this set? Is this unset for the delete case?

Reply:
As this feature does not support destroyStack. I just removed this.